collections/DBFilter.js

```diff
@@ -0,0 +1,27 @@
+Meteor.methods({
+  getFilter: function(date, days) {
```

Comment:
So you are using a method to retrieve the DB filter? Can you explain what the benefit of this will be?